src/tracer/tracer.ts

```diff
@@ -225,20 +234,73 @@ export class AutoblocksTracer {
     });
   }
 
-  public async sendEvent(message: string, args?: SendEventArgs): Promise<void> {
-    try {
-      // This store should only be set in the context of a test run
-      const store = testCaseRunAsyncLocalStorage.getStore();
-      if (store) {
-        await this.sendTestEventUnsafe(message, args);
-      } else {
-        await this.sendEventUnsafe(message, args);
-      }
-    } catch (err) {
-      if (readEnv(AutoblocksEnvVar.AUTOBLOCKS_TRACER_THROW_ON_ERROR) === '1') {
-        throw err;
-      }
-      console.error(`Error sending event to Autoblocks: ${err}`);
+  private async asyncSendEvent(
+    message: string,
+    args?: SendEventArgs,
+  ): Promise<void> {
+    // This store should only be set in the context of a test run
+    const store = testCaseRunAsyncLocalStorage.getStore();
+    if (store) {
+      await this.sendTestEventUnsafe(message, args);
+    } else {
+      await this.sendEventUnsafe(message, args);
     }
   }
+
+  public sendEvent(message: string, args?: SendEventArgs): void {
+    const task: BackgroundTask = {};
+    backgroundTasks.add(task);
+
```

Comment:
i think we might want to set the `timestamp` immediately here instead of within the `setImmediate` callback so it's the time you called `sendEvent` and not the time the callback was picked up by the event loop